.github/workflows/alpha-integration-tests.yaml

```diff
@@ -1,62 +1,57 @@
 name: DataRepo Alpha Integration Tests
 env:
-  GOOGLE_APPLICATION_CREDENTIALS: /tmp/jade-dev-account.json
-  GOOGLE_SA_CERT: /tmp/jade-dev-account.pem
+  K8_CLUSTER: jade-master-us-central1
+  GOOGLE_APPLICATION_CREDENTIALS: /tmp/alpha-test-runner.json
   GOOGLE_CLOUD_PROJECT: terra-datarepo-alpha
   GOOGLE_CLOUD_DATA_PROJECT: terra-datarepo-alpha-data
-  IT_JADE_API_URL: https://data.alpha.envs-terra.bio
   TDR_LOG_APPENDER: Console-Standard
-  PGHOST: 127.0.0.1
-  PGPASSWORD: postgres
-  DB_DATAREPO_URI: "jdbc:postgresql://${PGHOST}:5432/datarepo"
-  DB_STAIRWAY_URI: "jdbc:postgresql://${PGHOST}:5432/stairway"
 on:
   # Jenkins will kick this job off manually after Alpha is deployed
   workflow_dispatch: {}
 jobs:
   alpha-integration-test:
```

Comment:
Maybe call this `test-runner-alpha` for consistency?

Reply:
Also would it make sense to move some of the common smoke test code into a separate action (not sure if that's the right term here) to reduce repetition?

Reply:
I don't want to rename b/c devOps already triggers the run based on the name of the file. This is also going to include a few more steps in a follow-on PR, so I will bring up the topic of renaming the action to the team then!

We've really struggled with how to organize the action code - I'd be interested in hearing your ideas for how to improve it! Just for some background: We started off with a separate repo (datarepo-actions) and put common code in there. However, we've found that it painful to work with a separate repo, so we've moved away from using it.  I've also found that separate workflows within jade-data-repo to be confusing, especially when they're never used as a stand-alone action. So, the I've found the most clarity in keeping the bulk of the code in the individualized actions. With this methodology, you can actually see all of the steps that are being taken and when you expect the step in the Github UI, you can see the code for the action step. BUT, it means we end up with a lot of repeated code.

Reply:
That makes sense! Thanks for the detailed explanation!

Reply:
If code sharing of the bash code (run blocks) in a GitHub action is needed, would moving the code into scripts in the repo help? This is something we tried to do with Jenkins jobs. It also made it easier to test changes out locally since you could run the script called from the job.